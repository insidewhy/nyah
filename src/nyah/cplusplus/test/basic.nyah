grammar 'basic'

Thought = wherearemybricks
Guns    = notthekitten
Navaro  = ohplease
Global  = bum
