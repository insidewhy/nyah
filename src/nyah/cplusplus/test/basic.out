tuple("basic", vector(
    tuple("Thought", "wherearemybricks")
    tuple("Guns", "notthekitten")
    tuple("Navaro", "ohplease")
    tuple("Global", "bum")
))
